doc/multimc.1.txt

```diff
@@ -5,31 +5,31 @@ MULTIMC(1)
 
 NAME
 ----
-multimc - a launcher and instance manager for Minecraft.
+ultimmc - a launcher and instance manager for Minecraft.
 
 
 SYNOPSIS
 --------
-*multimc* ['OPTIONS']
+*ultimmc* ['OPTIONS']
 
 
 DESCRIPTION
 -----------
-MultiMC is a custom launcher for Minecraft that allows you to easily manage
+UltimMC is a custom launcher for Minecraft that allows you to easily manage
```

Comment:
Add here note about light-fork of MultiMC with cracked accounts and accounts from ely.by